matrix/Euler.hpp

```diff
@@ -48,23 +48,13 @@ class Euler : public Vector<Type, 3>
 
     Euler(const Dcm<Type> & dcm) : Vector<Type, 3>()
```

Comment:
Maybe we need to have a single a double precision implementation.
Since the type for the dcm is either going to be float or double, this should be pretty easy to do with the templates.